### PR TITLE
Feature/mmt 1490 update ummc from v1.10 to v1.12

### PR DESCRIPTION
**Pull request by htranho**

### CODE REVIEW

File: app/helpers/drafts_helper.rb [hunk 1]
```diff
@@ -398,6 +564,74 @@ module DraftsHelper
     ['Home Page', 'HOME PAGE']
   ]
   URLSubtypeOptions = [
```
Comment (htranho) [inline app/helpers/drafts_helper.rb]: use UMMC

File: spec/features/collection_drafts/forms/related_urls_form_spec.rb [hunk 2]
```diff
@@ -38,17 +38,9 @@
       within '.multiple.related-urls> .multiple-item-1' do
         expect(page).to have_field('Description', with: 'Example Description 2')
         expect(page).to have_field('URL Content Type', with: 'DistributionURL')
-        expect(page).to have_field('Type', with: 'GET SERVICE')
-        expect(page).to have_field('Subtype', with: 'DIF')
+        expect(page).to have_field('Type', with: 'GET DATA')
+        expect(page).to have_field('Subtype', with: 'DIRECT DOWNLOAD')
         expect(page).to have_field('URL', with: 'https://example.com/')
-
```
Comment (htranho) [inline spec/features/collection_drafts/forms/related_urls_form_spec.rb]: add structure of get data

File: spec/features/previews/related_urls_preview_spec.rb [hunk 3]
```diff
@@ -39,6 +39,7 @@
               expect(page).to have_content('Distribution URL')
             end
             within '.card-body' do
+              save_screenshot
```
Comment (htranho) [inline spec/features/previews/related_urls_preview_spec.rb]: remove this

File: app/helpers/drafts_helper.rb [hunk 4]
```diff
@@ -212,7 +212,7 @@ module DraftsHelper
     ['WRS-1'],
     ['WRS-2']
   ]
-  url_content_type_map = {
+  url_content_type_map_for_umms = {
```
Comment (charleshuang80) [inline app/helpers/drafts_helper.rb]: can we make this `url_content_type_map_for_umm_s`?

**Pull request merged**